shared/bundle_analysis/parsers/v2.py

```diff
@@ -110,20 +110,6 @@ def parse(self, path: str) -> int:
                 for event in ijson.parse(f):
                     self._parse_event(event)
 
-                if self.asset_list:
-                    insert_asset = Asset.__table__.insert().values(self.asset_list)
-                    self.db_session.execute(insert_asset)
-
-                if self.chunk_list:
-                    insert_chunks = Chunk.__table__.insert().values(self.chunk_list)
-                    self.db_session.execute(insert_chunks)
-
-                if self.module_list:
-                    insert_modules = Module.__table__.insert().values(self.module_list)
-                    self.db_session.execute(insert_modules)
-
-                self.db_session.flush()
-
                 # Delete old session/asset/chunk/module with the same bundle name if applicable
```

Comment:
Delete the old bundle (and its assets/chunks/modules) first before inserting new ones, this fixes a bug relating to bundle caching.